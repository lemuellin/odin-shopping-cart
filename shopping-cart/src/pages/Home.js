import '../style/home.css';

const Home = () => {
    return (
      <div className='home'>
        <div className='homeText'>
          <h3>keep it</h3>
          <h2>simple.</h2>
        </div>
      </div>
    );
  }
  
  export default Home;