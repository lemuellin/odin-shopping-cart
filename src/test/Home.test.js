import React from "react";
import { render } from "@testing-library/react";
import Home from '../pages/Home';

describe("Home", () => {
    it("Match Snapshot", () => {
        const { container } = render (<Home/>);
        expect(container).toMatchSnapshot();
    });

});