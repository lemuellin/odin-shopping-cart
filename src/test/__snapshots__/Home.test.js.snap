// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Match Snapshot 1`] = `
<div>
  <div
    class="home"
  >
    <div
      class="homeText"
    >
      <h3>
        keep it
      </h3>
      <h2>
        simple.
      </h2>
    </div>
  </div>
</div>
`;
